src/NServiceBus.Transport.RabbitMQ.TransportTests/When_customizing_outgoing_messages.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace NServiceBus.Transport.RabbitMQ.TransportTests;
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using global::RabbitMQ.Client;
+using global::RabbitMQ.Client.Events;
+using NServiceBus.TransportTests;
+using NUnit.Framework;
+
+public class When_customizing_outgoing_messages : NServiceBusTransportTest
```

Comment:
For my own benefit. My thought was that this is more of a user-facing option now so intuitively I would have written an acceptance test. In the acceptance test you also have the possibility to configure the transport at the level of the endpoint, which means you don't have to have the awkward coupling that requires a comment to explain why the customization is added in the configure class. 

Curious to hear why you did this as a transport test

Reply:
I guess it was the easiest option but I think you are right. I will re-write it as an acceptance test.